zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/actuator/PrometheusActuator.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.qa.util.actuator;
+
+import feign.Feign;
+import feign.Headers;
+import feign.RequestLine;
+import feign.Response;
+import feign.Retryer;
+import feign.Target.HardCodedTarget;
+import feign.jackson.JacksonDecoder;
+import feign.jackson.JacksonEncoder;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+
+@SuppressWarnings({"unused", "UnusedReturnValue"})
+public interface PrometheusActuator {
+
+  static PrometheusActuator of(final TestStandaloneBroker node) {
+    return of(node.actuatorUri("prometheus").toString());
+  }
+
+  static io.camunda.zeebe.qa.util.actuator.PrometheusActuator of(final String endpoint) {
```

Comment:
:x: Not sure why this is a fully qualified name. Can't we shorten it to just `PrometheusActuator`?